webapp/channels/src/components/pricing_modal/content.tsx

```diff
@@ -326,81 +237,28 @@ function Content(props: ContentProps) {
                 </button>
             </Modal.Header>
             <Modal.Body>
-                {!cloudFreeDeprecated && (
-                    <div className='pricing-options-container'>
-                        <div className='alert-option-container'>
-                            <div className='alert-option'>
-                                <span>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</span>
-                                <ExternalLink
-                                    onClick={() =>
-                                        trackEvent(
-                                            TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
-                                            'click_looking_to_self_host',
-                                        )
-                                    }
-                                    href={CloudLinks.DEPLOYMENT_OPTIONS}
-                                    location='pricing_modal_content'
-                                >{formatMessage({id: 'pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</ExternalLink>
-                            </div>
+                <div className='pricing-options-container'>
+                    <div className='alert-option-container'>
+                        <div className='alert-option'>
+                            <span>{formatMessage({id: 'pricing_modal.lookingToSelfHost', defaultMessage: 'Looking to self-host?'})}</span>
+                            <ExternalLink
+                                onClick={() =>
+                                    trackEvent(
+                                        TELEMETRY_CATEGORIES.CLOUD_PURCHASING,
+                                        'click_looking_to_self_host',
+                                    )
+                                }
+                                href={CloudLinks.DEPLOYMENT_OPTIONS}
+                                location='pricing_modal_content'
+                            >{formatMessage({id: 'pricing_modal.reviewDeploymentOptions', defaultMessage: 'Review deployment options'})}</ExternalLink>
                         </div>
                     </div>
-                )}
+                </div>
 
                 <div
                     className='PricingModal__body'
-                    style={{marginTop: cloudFreeDeprecated ? '74px' : ''}}
+                    style={{marginTop: ''}}
                 >
-                    {!cloudFreeDeprecated && (
```

Comment:
I am a bit lost here. In this file, I feel we are going on the basis of "`cloudFreeDeprecated` is always false". So... in this case, this card should show, right?
Maybe this card shouldn't show for other reasons, but wanted to verify.